baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -105,7 +112,10 @@ def handle(self, message: kombu.Message) -> None:
                 "Unhandled error while trying to process a message.  The message "
                 "has been returned to the queue broker."
             )
-            message.requeue()
+            if self.error_handler_fn:
+                self.error_handler_fn(context, message.decode(), message)
```

Comment:
what happens if decoding was the problem in the first place? would it make sense to re-use the decoded value from the happy path here and default it to something like `None` should the failure have happened before decoding was complete?